src/wgquickprocess.cpp

```diff
@@ -21,11 +21,10 @@ Logger logger(
 }  // namespace
 
 // static
-bool WgQuickProcess::createConfigFile(const QString& outputFile,
-                                      const InterfaceConfig& config,
-                                      const QMap<QString, QString>& extra) {
+QVariant WgQuickProcess::createConfigString(
```

Comment:
The use of QVariant here is kind-of unnecessary, we can check for a failed config by testing `QString::isNull()` or `QString::isEmpty()`